.gitignore

```diff
@@ -6,3 +6,6 @@ mkosi.builddir/
 mkosi.local.conf
 mkosi.crt
 mkosi.key
+
+# Ignore Python venvs for local installation of mkosi
+.venv
```

Comment:
For completeness I'd add `venv` (without the leading dot) as well. I always thought hiding the venv went a bit against the explicit is better than implicit.

Reply:
I prefer the .venv as dotted files are commonly excluded from indexing by daemons like baloo (which is already quite annoying configuring them to exclude all the mkosi directories)